<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModuleDialog</class>
 <widget class="QDialog" name="ModuleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="typeLabel">
     <property name="text">
      <string>Node type</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="typeBox">
     <item>
      <property name="text">
       <string>Dicio</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Planalta</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sylvatica</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="idBox">
     <property name="text">
      <string>Automatic ID</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="idLabel">
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="idEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="altLabel">
     <property name="text">
      <string>Alt name</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="altEdit"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="okBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okBox</sender>
   <signal>accepted()</signal>
   <receiver>ModuleDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okBox</sender>
   <signal>rejected()</signal>
   <receiver>ModuleDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
